actionview/lib/action_view/helpers/form_options_helper.rb

```diff
@@ -137,24 +137,35 @@ module FormOptionsHelper
       #
       # ==== Gotcha
       #
-      # The HTML specification says when +multiple+ parameter passed to select and all options got deselected
-      # web browsers do not send any value to server. Unfortunately this introduces a gotcha:
-      # if a +User+ model has many +roles+ and have +role_ids+ accessor, and in the form that edits roles of the user
-      # the user deselects all roles from +role_ids+ multiple select box, no +role_ids+ parameter is sent. So,
-      # any mass-assignment idiom like
+      # The HTML specification says when the +multiple+ attribute is specified on a select element and all options
+      # are deselected, web browsers do not send any value to the server. Unfortunately, this introduces a gotcha.
+      # For example, given a multiple select in a form that edits people associated with a post:
       #
-      #   @user.update(params[:user])
+      #   select :post, :person_ids, Person.all.collect { |p| [ p.name, p.id ] }, { multiple: true })
+      #
+      # would become:
       #
-      # wouldn't update roles.
+      #   <select name="post[person_ids]" id="post_person_ids" multiple="true">
```

Comment:
I just tested this locally and this is missing the extra `[]` in the name:
```suggestion
      #   <select name="post[person_ids][]" id="post_person_ids" multiple="true">
```

Reply:
Great catch. Thank you! This is updated as well.  Thanks for your time reviewing this and for all the great feedback!